Patterns/Korean/Korean-Numbers.yaml

```diff
@@ -302,23 +302,69 @@ SimpleIntegerPercentageRegex: !nestedRegex
   references: [NegativeNumberTermsRegexNum, ZeroToNineFullHalfRegex, AllIntRegex, RoundNumberIntegerRegex, NegativeNumberTermsRegex]
 #NumberRangeExtractor
 TillRegex: !simpleRegex
-  def: (부터|까지|--|-|—|——|~)
+  def: (부터|에서|--|-|—|–|——|~)
 MoreRegex: !simpleRegex
-  def: (초과|많|높|크|더많|더높|더크|>)
+  def: (초과|많|높|크|더많|더높|더크|넘는|초과이다|크고)
 LessRegex: !simpleRegex
-  def: (미만|적|낮|작|더적|더낮|더적|<)
+  def: (미만|적|낮|작|더적|더낮|더적|이하|이하이다|<|아래|작다)
 EqualRegex: !simpleRegex
-  def: (동일|같|=)
+  def: (동일|같|=|(해당하는)|는|그와 같다)
+RangePrefixLessRegex: !simpleRegex
+  def: (까지최소|(?<!>|=)<|≤)
+RangePrefixMoreRegex: !simpleRegex
+  def: ((?<!<|=)>|≥)
 MoreOrEqual: !nestedRegex
   def: (({MoreRegex}\s*(거나)?\s*{EqualRegex}))
   references: [ MoreRegex, EqualRegex ]
 MoreOrEqualSuffix: !simpleRegex
   def: \s*(이상)
 LessOrEqual: !nestedRegex
-  def: (({LessRegex}\s*(거나)?\s*{EqualRegex}))
+  def: (?:(이|보다)?)?\s*(({LessRegex}\s*(거나)?\s*{EqualRegex}(은|다)?))
   references: [ LessRegex, EqualRegex ]
 LessOrEqualSuffix: !simpleRegex
-  def: \s*(이상)
+  def: \s*(이하)
+OneNumberRangeMoreSeparateRegex: !simpleRegex
+  # TODO: modify below regex according to the counterpart in English
```

Comment:
If these first two are not necessary, change the TODO comment to "# Regex not applicable in Korean".
If they may be written later, you can leave as is for now.

Reply:
Sure, we'll update the TODO comment.